### ISSUE TITLE
How to use getStats with Request()

### ISSUE THREAD

**Issue by carun** [ikod/dlang-requests#87]
`HTTPRequest` had `getStats` function. How do we get a similar statistics with `Request()`. Any examples would be helpful.

**Comment by ikod** [comment]
Hello, @carun 

Will look into this. If I missed it during transition to version 1.0.0 - then I'll try to restore it.

Thanks for your request

**Comment by carun** [comment]
@ikod Thanks for entertaining the request. Much appreciate you maintaining this library!

**Comment by ikod** [comment]
Hello @carun 

You can try last commit - I moved getStats from HTTPResponse to Response, so something like
```
    auto rs = rq.get("https://google.com/");                                                                                     
    writeln(rs.getStats);                                                                                                   

```
should work.
I will roll out new release as soon as you confirm that it works for you.

> Much appreciate you maintaining this library!

Thanks! Its pure pleasure :)

**Comment by carun** [comment]
Sure, let me test it right away!

**Comment by ikod** [comment]
Hello, @carun 

Fixed with release v1.0.4

Thanks for report!
